src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -34,20 +42,26 @@ public void should_return_a_404_if_task_does_not_exist() {
             .body("error", equalTo(HttpStatus.NOT_FOUND.getReasonPhrase()))
             .body("status", equalTo(HttpStatus.NOT_FOUND.value()))
             .body("message", equalTo(String.format(
-                "There was a problem updating the task with id: %s. The task could not be found.",
+                LOG_MSG_THERE_WAS_A_PROBLEM_FETCHING_THE_VARIABLES_FOR_TASK,
                 nonExistentTaskId
             )));
     }
 
+    private String getAssigneeId(Header caseworkerBAuthorizationOnly) {
+        return authorizationHeadersProvider.getUserInfo(caseworkerBAuthorizationOnly.getValue()).getUid();
+    }
+
     @Test
     public void should_return_a_204_when_assigning_a_task_by_id() {
-
-
         Map<String, String> task = common.setupTaskAndRetrieveIds();
 
+        String assigneeId = getAssigneeId(authorizationHeadersProvider.getCaseworkerBAuthorizationOnly());
```

Comment:
Small suggestion if possible, could we add the permission for the user types as a comment in the section that defines it role assignment properties? Would make it easier to qualify the AC by permission. 

for example; AuthorizationHeadersProvider class getCaseworkerBAuthorizationOnly() has 
/*
         * Role assignment Properties:
         * - Organizational case role with:
         * --classification = PUBLIC
         * --jurisdiction = IA
         */
Is it possible to add a new line "*--MANAGE etc"?

Reply:
done

Reply:
done